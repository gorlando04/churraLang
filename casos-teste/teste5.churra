amigo "Jo√£o" contribui 80
amigo "Carlos" contribui 90 vegetariano
item bebida preco_L 10 quantidade_L 2
item comida preco_unit 30 quantidade 1
item entretenimento preco_unit 20 quantidade 1
rateio igual