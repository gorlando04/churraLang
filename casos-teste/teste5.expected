Total do churrasco: R$70.00
Carlos deve pagar: R$35.00
JoÃ£o deve pagar: R$35.00